### PR TITLE
PP-4463 Add StripePerson model object for use with Stripe client

### PR DESCRIPTION
**Pull request by alexbishop1**
Add `StripePerson` model object, which represents the details about a responsible person in the format that the Stripe Connect Persons API expects.

### CODE REVIEW

File: app/services/clients/stripe/stripePerson.model.test.js [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+      address_line1: addressLine1,
+      address_city: addressCity,
+      address_postcode: addressPostcode,
+      dob_day: dobDay,
+      dob_month: 0,
+      dob_year: dobYear
+    })).to.throw('StripePerson "dob_month" must be larger than or equal to 1')
+  })
+
+  it('should fail when month is less than 1000', () => {
```
Comment (DanailMinchev) [inline app/services/clients/stripe/stripePerson.model.test.js]: probably copy pasta `should fail when year is less than 1000` ?

Comment (alexbishop1) [reply app/services/clients/stripe/stripePerson.model.test.js]: I never copy and paste, Dan. I type everything out by hand. Your eyes are deceiving you.

Comment (alexbishop1) [reply app/services/clients/stripe/stripePerson.model.test.js]: No, really, should be fixed now and also added a test for the month being too large.

Comment (DanailMinchev) [reply app/services/clients/stripe/stripePerson.model.test.js]: :D

**Pull request merged**